plugins/python-build/scripts/add_miniconda.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python3.7
```

Comment:
It would be nice if there's some usage example of this script. In fact, just running the script without giving any arguments will let the script to update the python-build's build definitions for miniconda based on the contents of Miniconda's download page, however, it's not very obvious without looking into the script itself. Since this is just some sort of utility script for maintainers, it doesn't need to be some individual documentation file, however, it's nice if there's some practical usage example is in somewhere (e.g. show message on `--help`, etc.)

Reply:
Yep - I think at time of writing 3.8 wasn't out, and 3.7 was pretty new so I wanted to be explicit about it. Now I'm happy to just specify 3 in the shebang and document that it's 3.7+.

Usage example is a good idea. I'll write a module docstring and have that reported by the help message.